WebARonARKit/WebARonARKit/ViewController.m

```diff
@@ -628,7 +678,7 @@ - (void)session:(ARSession *)session didUpdateFrame:(ARFrame *)frame
                          pProjectionMatrix[10], pProjectionMatrix[11],
                          pProjectionMatrix[12], pProjectionMatrix[13],
                          pProjectionMatrix[14], pProjectionMatrix[15],
-                         anchors];
+                         anchorsStr];
```

Comment:
Would not be better to handle the update or the anchors in JS so this list does not have to be passed per frame? My initial idea was to just dispatch the events and keep the up to date structure/container of anchors in the JS side. When the getPlanes call is made, just return the up to date structure. This seems less efficient (bigger string every frame) and more garbage collection generated in the JS side.

Reply:
ok, i'll do that. give me a couple minutes.